src/test/java/qna/domain/AnswerRepositoryTest.java

```diff
@@ -27,29 +26,30 @@ class AnswerRepositoryTest {
     @Autowired
     private QuestionRepository questionRepository;
 
+    private User user;
+
+    private Question question;
+
     @BeforeEach
     void setUp() {
```

Comment:
```suggestion
    void setUp(@Autowired final AnswerRepository answerRepository) {
```

소소한 팁을 드리자면 setUp에서도 빈 주입을 받을 수 있습니다~